### DISCUSSION TITLE
field names of the filter is always in lowercase

### DISCUSSION THREAD

**Discussion by TestUE4byNurlan** [Adrinalin4ik/Nestjs-Graphql-Tools#15] [General]
If the database has a field with uppercase char and you have some input with uppercase as well,
then the filter (Brackets) object returns the fields in lowercase and the database throws an error. did you have such problems with typeorm and postgresdb?

**Comment by Adrinalin4ik** [comment]
Hey! No, I don't. It was mentioned in the description that you have to use snake case fields in the db. Postgres doesn't like camel case.
